### PR TITLE
Add ability to loop around when reaching either end

### PR DESCRIPTION
**Pull request by julbra**
Note that this combines well with autoplay to give us an infinite loop. Which should address #12 in a more general way than https://github.com/SSENSE/vue-carousel/pull/43

### CODE REVIEW

File: tests/client/components/__snapshots__/carousel.spec.js.snap [hunk 1]
```diff
@@ -48,16 +48,16 @@ exports[`Carousel should begin autoplaying when option specified 1`] = `
...
     .VueCarousel-slide
     .VueCarousel-slide
 |  
 .VueCarousel-pagination
   .VueCarousel-dot-container
-    .VueCarousel-dot(style=\'margin-top: 20px; padding: 10px;\')
-      .VueCarousel-dot-inner(style=\'width: 10px; height: 10px; background: rgb(239, 239, 239);\')
-    .VueCarousel-dot.VueCarousel-dot--active(style=\'margin-top: 20px; padding: 10px;\')
-      .VueCarousel-dot-inner(style=\'width: 10px; height: 10px; background: rgb(0, 0, 0);\')
+    .VueCarousel-dot(style='margin-top: 20px; padding: 10px;')
```
Comment (julbra) [inline tests/client/components/__snapshots__/carousel.spec.js.snap]: Not sure why these were reformatted as this is generated

**Comment by julbra** [comment]
Could anyone review this? Let me know if I can help further.

**Comment by julbra** [comment]
@coolhatharry I have added some documentation and a jsfiddle example in the doc/source. How does the public submodule get updated?

**Comment by iamdubx** [comment]
Please add this feature!

**Comment by realtebo** [comment]
So this feature is available into master? Hiow can  I specify to use master using composer dependency specification?

**Comment by iamdubx** [comment]
@realtebo just add `loop`! Actually this feature is already works, need to update Docs.

**Comment by realtebo** [comment]
Oh.. Yes 
it's working. Thanks a lot

**Comment by iamdubx** [comment]
I guess I know why you don't add yet loop in docs. It has weird behavior. I think, I will not use loop for now.

**Comment by imbolc** [comment]
Is it possible to make loop moving in one side without any rewinds at the end?

**Pull request merged**